pkg/clientset/tiltapi/wire.go

```diff
@@ -0,0 +1,19 @@
+package tiltapi
+
+import (
+	"fmt"
+
+	"k8s.io/client-go/rest"
+
+	"github.com/tilt-dev/tilt/pkg/model"
+)
+
+func ProvideRESTConfig(apiserverHost model.WebHost, apiserverPort model.WebPort) *rest.Config {
```

Comment:
have you seen https://github.com/tilt-dev/tilt/blob/master/internal/hud/server/apiserver.go#L78 ? it also works for cases where there's an in-memory connection

Reply:
Ah yep, this is vestigial - this code has gone through one too many rebases 😝